openquakeplatform_ipt/views.py

```diff
@@ -186,11 +186,19 @@ def taxonomy_mapping_check(full_path):
                 return (False, "'%s' field not found" % field)
 
         sum = 0.0
```

Comment:
`sum` is a builtin, please use a different name

Reply:
Done